src/lib/util/create-link.ts

```diff
@@ -0,0 +1,4 @@
+export function createLink(url: string, {assignee, secret}){
```

Comment:
I decided not to pass callback that change showAlert variable, because it seems that it is more useful to do it in copyLink. It also make this external function less reusable on my opinion..

Reply:
That's sound good. Let's give the function a more accurate name. So it's writes to the clipboard. So let's use that as our guide.

Reply:
Let's also add the correct typing for the props and what it returns